wolfhsm/wh_comm.h

```diff
@@ -69,7 +81,7 @@ enum WH_COMM_ENUM {
     (_magic                 & WH_COMM_MAGIC_ENDIAN_MASK) ==  \
     (WH_COMM_MAGIC_NATIVE   & WH_COMM_MAGIC_ENDIAN_MASK)
 
-/* Header for a packet, request or response. On-the-wire format */
+/* 8 byte Header for a packet, request or response. On-the-wire format */
```

Comment:
I want static asserts :(

Reply:
We should add this sizeof() check to the check padding.  prolly lots more too.